src/core_plugins/kibana/index.js

```diff
@@ -17,7 +17,8 @@ module.exports = function (kibana) {
       return Joi.object({
         enabled: Joi.boolean().default(true),
         defaultAppId: Joi.string().default('discover'),
-        index: Joi.string().default('.kibana')
+        index: Joi.string().default('.kibana'),
+        indexPatternPlaceholder: Joi.string().default('logstash-*')
```

Comment:
I understand why you need this to be a configuration in the kibana.yml file, given the "on load" condition you're trying to solve for here, but as I mentioned, I don't think that's the right solution. The page is fundamentally broken, and introducing a config value we later have to deprecate or remove isn't desirable. I'd like to keep the scope of this PR to just the advanced setting, and see this removed.